local-helm/buyingcatalogue/charts/ordapi-db/templates/deployment.yaml

```diff
@@ -52,7 +42,7 @@ spec:
             - name: ACCEPT_EULA
               value: "Y"      
             - name: MSSQL_PID
-              value: Developer  
+              value: Express  
```

Comment:
I think you should leave this as developer. AFAIK, this unlocks more features and makes it more like the cloud env.

Reply:
👍